src/multiselect.js

```diff
@@ -373,6 +379,31 @@ export class Multiselect {
     }
   }
 
+  /**
+   * Handle a keyboard navigation key-down on the workspace.
+   * @param {KeyboardEvent} e The keyboard event.
+   * @private
+   */
+  unbindMultiselectCopyPaste_(e) {
+    // TODO: Update this to re-register/unregiser the original shortcuts after
+    //  Blockly/keyboard navigation plugin update
+    // This is to unregister the multiselect plugin's shortcuts when the user is in
+    // the keyboard navigation mode. Currently, when the user is in keyboard
+    // accessibility mode, they cannot use the normal copy/cut/paste functionalities.
+    // This is because the original (Blockly core) copy/cut/paste functions do not
+    // allow for collisions. This can be fixed either by allowing for collisions in
+    // the Blockly core copy/cut/paste functions or allowing for unregister/re-registering
+    // of the keyboard navigation plugin's copy/cut/paste functions.
+    if (this.keyboardNavKeys_.indexOf(e.key.toLocaleLowerCase() > -1) &&
```

Comment:
Looks like we have a bracket issue here. Shouldn't this be:
```suggestion
    if (this.keyboardNavKeys_.indexOf(e.key.toLocaleLowerCase()) > -1 &&
```

?

So I wonder if you have already tested the code, is this still working 100% Okay even if you write it in this way? (as I guess this then will never get triggered)

And by the way, the code means that, if any of the key (i.e. 'ctrl', 'shift', 'k') is pressed (instead of all of them pressed all together), this will be true, is this Okay and reasonable?

Reply:
It still works even though there was a typo. The code checks if any of the keys (i.e. 'ctrl', 'shift', 'k') are pressed AND the keyboard accessibility mode is on AND if the multiselect shortcuts have been registered. This is reasonable as the first if-block will only fire when the keyboard navigation plugin is enabled (one of the 3 keys are pressed, keyboard accessibility mode is turned on, and multiselect shortcuts are currently registered). The else-if block will fire when the keyboard accessibility mode is turned off and the multiselect shortcuts are not registered.

Reply:
Now this gets interesting, since it still works under typo, and `(this.keyboardNavKeys_.indexOf(e.key.toLocaleLowerCase() > -1)` should always return `-1`, while `-1 && true` is always true, which indicates that the check here is not needed at all if you have already fully tested this before.

So why do we still have this `one of the 3 keys are pressed` if you have already fully tested the code with this typo? This also removes the overhead for setting these keys as those are configurable.

Reply:
Ah, I see now. I just tested the code without the keyboardNavKeys and it seems to work. I will push the latest changes now.